src/main/java/Main.java

```diff
@@ -67,6 +75,21 @@ public static void main(String[] args) {
         DisplayTaskTreeInputBoundary displayTaskTreeInteractor = new DisplayTaskTreeInteractor(displayTaskTreePresenter);
         DisplayTaskTreeController displayTaskTreeController = new DisplayTaskTreeController(displayTaskTreeInteractor);
         TaskTreeUI taskTreeUI = new TaskTreeUI(windows, displayTaskTreeController);
+        //createCurriculumPresenter.
+
+        DisplayAvailabilityTimeBlockPresenter availabilityPresenter = new DisplayAvailabilityTimeBlockPresenter(new ArrayList<>());
+        DisplayAvailabilityTimeBlockInteractor availabilityInteractor = new DisplayAvailabilityTimeBlockInteractor(availabilityPresenter);
+        DisplayAvailabilityTimeBlockController controller = new DisplayAvailabilityTimeBlockController(availabilityInteractor);
+        DisplayAvailabilityTimeBlockUI view = new DisplayAvailabilityTimeBlockUI(windows, controller);
+        availabilityPresenter.addAvailabilityObserver(view);
+
+        SetAvailabilityPresenter setAvailabilityPresenter = new SetAvailabilityPresenter(new ArrayList<>());
+        SetAvailabilityUseCase setAvailabilityUseCase = new SetAvailabilityUseCase(setAvailabilityPresenter);
+        SetAvailabilityController setAvailabilityController = new SetAvailabilityController(setAvailabilityUseCase);
+        SetAvailabilityUI setAvailabilitViewUI = new SetAvailabilityUI(windows, setAvailabilityController);
+        setAvailabilityPresenter.addViewObserver(setAvailabilitViewUI);
```

Comment:
Excellent work with your dependency inversion here! This will allow us to make sure we are adhering to clean architecture while also sticking to the SRP.